pyproject.toml

```diff
@@ -1,8 +1,8 @@
 # SPDX-License-Identifier: MIT
 
 [build-system]
-requires = ["setuptools>=61"]
-build-backend = "setuptools.build_meta"
+requires = ["pdm-backend>=2.0"]
```

Comment:
Is it safe to depend on a minimum version here, instead of `~=`?

Reply:
This is a minimum version here

Reply:
Yeah, and I'm not sure if that's a good idea - maybe `pdm-backend~=2.0` would be a better choice